assets/in

```diff
@@ -23,6 +23,8 @@ use_v4="$(echo "${payload}" | jq -r '.source.use_v4 // ""')"
 path=$(echo "${payload}" | jq -r '.source.path // ""')
 suffix=$(echo "${payload}" | jq -r '.source.suffix // ""')
 encryption_key="$(echo "${payload}" | jq -r '.source.encryption_key // ""')"
+skip_ssl_verification="$(echo "$payload" | jq -r '.source.skip_ssl_verification // ""')"
+ca_bundle="$(echo "$payload" | jq -r '.source.skip_ssl_verification // ""')"
```

Comment:
Same here.. ca_bundle is being set as source.skip_ssl_validation.

Reply:
Fixed.